### PR TITLE
Add v1.0 documentation for Azure AD named locations and security defaults API

### PR DESCRIPTION
**Pull request by abhishekkp09**

### CODE REVIEW

File: api-reference/v1.0/toc.yml [hunk 1]
```diff
@@ -3773,6 +3815,13 @@ items:
             href: api/tokenlifetimepolicy-delete.md
           - name: List applied to applications
             href: api/tokenlifetimepolicy-list-appliesto.md
+        - name: Security defaults policy
```
Comment (davidmu1) [inline api-reference/v1.0/toc.yml:3818]: What is @dkershaw10 thinking about the location of this set of APIs? There has been some discussion about having all policies listed under a Policies node.

Comment (abhishekkp09) [reply api-reference/v1.0/toc.yml:3818]: It looks like they are already listed under a Policies node.

**Comment by videor** [comment]
@Lauragra ready to merge.

File: api-reference/v1.0/api/identitysecuritydefaultsenforcementpolicy-get.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
```
Comment (Lauragra) [inline api-reference/v1.0/api/identitysecuritydefaultsenforcementpolicy-get.md:1]: This and the following API topic are not included in the TOC. Please add them to the appropriate location.

**Comment by abhishekkp09** [comment]
> Please make sure that all method topics are exposed in the v1.0 TOC. Thanks.

Sorry I had added it at first but it got removed during rebase. Added it back.

**Pull request merged**